Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -1085,9 +1085,11 @@ void function ThreadedAuthAndConnectToServer( string password = "" )
 	}
 	else
 	{
+		var reason = NSGetAuthFailReason()
+
 		DialogData dialogData
 		dialogData.header = "#ERROR"
-		dialogData.message = "Authentication Failed"
+		dialogData.message = "" + reason
```

Comment:
```suggestion
		dialogData.message = reason
```

Reply:
It's apparently [untyped](https://github.com/R2Northstar/NorthstarMods/actions/runs/4744308144/jobs/8425040066).